docs/reference/api/python/driver.rst

```diff
@@ -19,6 +19,8 @@ tvm.driver
 ----------
 .. automodule:: tvm.driver
 
+.. automodule:: tvm.driver.tvmc
```

Comment:
Do you have an idea of what APIs this was supposed to add? Poking around the rendered docs in https://pr-docs.tlcpack.ai/PR-12705/3/docs/reference/api/python/index.html I can't find any  related to `tvm.driver.tvmc` aside from "TVMC - TVM driver command-line interface"

Reply:
Interesting. I wanted to see the main APIs `tvmc.tune|compile|run` in that at least.